models/sources/src_history_ledgers.yml

```diff
@@ -156,3 +156,23 @@ sources:
               - not_null:
                   config:
                     where: closed_at > current_timestamp - interval 2 day
+          - name: soroban_fee_write_1kb
+            description: '{{ doc("soroban_fee_write_1kb") }}'
+            tests:
+              - not_null:
+                  config:
+                    where: closed_at > current_timestamp - interval 2 day
```

Comment:
Unsure if these test config apply to these 3 params

Reply:
They apply to `node_id` and `signature`, but idk how `soroban_fee_write_1kb` is handled for old data. I think it would be aggressive for us to add a null check to that field.

Reply:
Got it. Gonna 🔪 it